src/main/java/org/myrobotlab/sensor/TimeEncoder.java

```diff
@@ -398,4 +401,56 @@ public void setPos(Double pos) {
     positions.setPosition(name, estimatedPos);
   }
 
+  @Override
+  public boolean hasInterface(Class<?> class1) {
+    Class<?>[] faces = TimeEncoder.class.getInterfaces();
+    for (Class<?> c : faces) {
+      if (c.equals(class1)) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  @Override
+  public boolean hasInterface(String interfaze) {
+    // TODO Auto-generated method stub
+    return false;
+  }
+
+  @Override
+  public boolean isType(Class<?> clazz) {
+    // TODO Auto-generated method stub
+    return false;
+  }
+
+  @Override
+  public boolean isType(String clazz) {
+    // TODO Auto-generated method stub
+    return false;
+  }
+
+  @Override
```

Comment:
remove these methods, they don't do anything

Reply:
They both implement Attachable.  Usually, this is an indication that this TimeEncoder and I2CBus should have been services. Not doing it in this pr.